dags/looker.py

```diff
@@ -67,6 +67,18 @@
     secret="airflow-gke-secrets",
     key="probe_scraper_secret__looker_api_client_secret_staging",
 )
+looker_client_id_prod = Secret(
+    deploy_type="env",
+    deploy_target="LOOKER_CLIENT_ID",
+    secret="airflow-gke-secrets",
+    key="probe_scraper_secret__looker_api_client_id_prod",
+)
+looker_client_secret_prod = Secret(
+    deploy_type="env",
+    deploy_target="LOOKER_CLIENT_SECRET",
+    secret="airflow-gke-secrets",
+    key="probe_scraper_secret__looker_api_client_secret_prod",
+)
```

Comment:
In the future it could be nice to change `lookml-generator` to expect these same environment variables rather than `LOOKER_API_CLIENT_ID` and `LOOKER_API_CLIENT_SECRET`.